addons/crm_claim/data/crm_claim_demo.xml

```diff
@@ -1,96 +1,94 @@
-<?xml version="1.0"?>
-<openerp>
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
     <data noupdate="1">
     
-    <!-- 
-        ((((((((((( Demo Cases ))))))))))) 
-    -->
+    <!--((((((((((( Demo Cases )))))))))))-->
     
         <record id="crm_claim_1" model="crm.claim">
-            <field eval="time.strftime('%Y-%m-04 10:45:36')" name="date"/>
+            <field name="date" eval="time.strftime('%Y-%m-04 10:45:36')"/>
             <field name="partner_id" ref="base.res_partner_3"/>
-            <field eval="&quot;1&quot;" name="priority"/>
+            <field name="priority" eval="&quot;1&quot;"/>
```

Comment:
May for string your can put something like(for better readability):
```<field name="priority">1</field>```
Applies to whole file